www/blak05/hw3/main.css

```diff
@@ -0,0 +1,97 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+}
+.red{
+    color:red;
+}
+.green{
+    color: rgb(0,255,0);
+}
+.blue{
+    color: #0000FF;
+    font-weight: bold;
+}
+.bigger-text{
+    font-size: 18px;
+}
+.even-bigger-text{
+    font-size: 24px;
+    font-style: italic;
+}
+.brown{
+    color:white;
+    background-color: brown;
+}
+p.center{
+    text-align: center;
+    line-height: 32px;
+}
+p.capitals{
+    text-transform: capitalize;
+    text-align: justify;
+}
+.hidden{
+    display: none;
+}
+.hidden-2{
+    visibility: hidden;
+}
+.box1{
+    width: 400px;
+    height: 300px;
+    border: 1px solid purple;
+    padding: 30px;
+}
+.box2{
+    width: 500px;
+    height: 200px;
+    background-color: red;
+    margin: 100px;
+}
+.box3{
+    height: 150px;
+    width: 100%;
```

Comment:
chybi box-sizing: border-box, proto se stalo, ze sirka tohohle boxu je vetsi nez sirka jeho containeru, i kdyz to ma byt 100% , protoze je tam navic padding a border...

box-sizing: border-box je dobre davat do wildcard selektoru (hvezdicka)